src/tokens/transformation/tailwind/formatCssVars.js

```diff
@@ -29,7 +29,9 @@ module.exports = ({ dictionary, options, file }) => {
       format: 'tailwind',
       dictionary: groupedTokens.rest,
       outputReferences
-    }).replace(/desktop-/gm, '') +
+    })
+      .replace(/desktop-/gm, '')
+      .replace(/browser-/gm, '') +
```

Comment:
Either now, or later, I believe that this needs to be refactored to better account for this type of removal. Maybe we should remove this kind of thing at the parser level?

Maybe even in Figma (if possible) since we no longer really need to distinguish different platforms (or I'm calling them layers to disambiguate from StyleDictionary's use of "platforms").